Cargo.lock

```diff
@@ -14,6 +14,7 @@ dependencies = [
  "num_cpus 1.7.0 (registry+https://github.com/rust-lang/crates.io-index)",
  "regex 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
  "regex-syntax 0.4.1 (registry+https://github.com/rust-lang/crates.io-index)",
+ "shell-escape 0.1.3 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Why does this PR re-introduce shell-escape? Is this needed somewhere?

Reply:
I don't actually remember adding that. It's not used for anything I'll remove that.